views/beers.hbs

```diff
@@ -0,0 +1,8 @@
+<div>
+    {{#each beersFromApi}}
+        <img src="{{this.image_url}}" alt="beers">
+        {{this.name}}
```

Comment:
And you need to pass the information that you are receiving, inside `<p>` or `<h1>` (paragraphs or heading tags).